src/_angular/scroll-setting/scroll-setting.html

```diff
@@ -22,9 +22,11 @@
     <div class="form-group">
       <label class="control-label">{{'scroll.speed.label' | translate}}</label>
       <select id="scroll-speed" ng-model="scroll.speed" class="form-control">
+        <option value="slowest">{{'scroll.speed.slowest' | translate}}</option>
```

Comment:
@pcsandford In this old commit https://github.com/Rise-Vision/widget-settings-ui-components/commit/f3bae47f450aa6c84cc9d787800de0bee2af6b4e#diff-dca669fb2752fb8f28473854fadd2fef , the "slowest" and "fastest" options used to exist and were removed. I'm concerned we're re-introducing options that should not be there as _I think_ there are widgets other than the Text widget that use this component. Have you been able to confirm re-introducing these options won't cause an issue in other Widgets if those other Widgets eventually upgrade to this version of settings-ui-components?

Reply:
@stulees good question, and probably the reverse could have been asked when those speeds were removed in terms of 'what would happen if the text widget were to upgrade to this version'. I have not tested the other widgets like RSS and spreadsheet but I suspect that if they upgraded to this version they would have those options re-introduced.

When discussing with @jungeunyoon I think it was decided to bring the text widget up to date with this version and then, as a separate task when required, update this repo so that the correct settings are applied to the different widgets. In any event, even without this commit we have the situation where the latest version of this repo contains settings that are not common across all widgets. 

Probably easiest to discuss on Tuesday at standup, so in the meantime have a great Canada Day weekend! :)

@santiagonoguez @sheadarlison FYI